lib/quby/questionnaires/entities/question.rb

```diff
@@ -166,9 +166,6 @@ def initialize(key, options = {})
             @validations << {type: :valid_float, explanation: options[:error_explanation]}
           elsif @type == :integer
             @validations << {type: :valid_integer, explanation: options[:error_explanation]}
-          elsif @type == :date
```

Comment:
Ik heb juist het liefst zoveel mogelijk code in subclasses. Doch wel liever met een nette add_validation method